api/barriers/serializers/data_workspace.py

```diff
@@ -513,3 +514,11 @@ def get_trade_direction(self, instance) -> typing.Optional[str]:
             )
         else:
             return None
+
+    def get_priority_level(self, instance) -> typing.Optional[str]:
+        if (
+            instance.priority_level == "NONE"
+            and instance.top_priority_status == "APPROVED"
```

Comment:
this should also include "REMOVAL_PENDING" and "RESOLVED" I think

```suggestion
            and instance.top_priority_status in ("APPROVED", "REMOVAL_PENDING", "RESOLVED")
```

Reply:
the ticket just said "APPROVED" state for top priority